### DISCUSSION TITLE
Plots that we have will be discarded after 17th?

### DISCUSSION THREAD

**Discussion by fjbotto** [Chia-Network/chia-blockchain#4076] [Q&A] [answered]
I'm asking because of [Brams announce](https://twitter.com/bramcohen/status/1390407484179763202?ref_src=twsrc%5Etfw)
of new pooling protocol

**Comment by Akaino** [comment, marked as answer]
Your plots still work for solo farming. If you want to join a pool you'd have to create new plots though.

**Reply by fjbotto** [reply]
Thanks for your answer

**Reply by unitedchia** [reply]
Exactly. The new plots will be portable though. So you will be able to switch pools at any time, if you want.

One of the upcoming pools is us [chiaunited.com](https://chiaunited.com). Feel free to visit and subscribe for news about our launch.
